plugins/qtosgrave/qtosgviewer.h

```diff
@@ -396,6 +396,7 @@ public slots:
 
     //@{ Message Queue
     list<GUIThreadFunctionPtr> _listGUIFunctions; ///< list of GUI functions that should be called in the viewer update thread. protected by _mutexGUIFunctions
+    list<GUIThreadFunctionPtr> _excessGUIFunctions; ///< list of GUI functions that exceed the queue limit and are thus stashed.
```

Comment:
`_excessGUIFunctions` => `_listGUIFunctionsBuffer`

in openrave you see variables like `vxxx` or `bxxx`, they are vector and boolean.